wicketd/src/artifacts.rs

```diff
@@ -37,14 +35,17 @@ use uuid::Uuid;
 
 use crate::installinator_progress::IprArtifactServer;
 
+mod extracted_artifacts;
+
+pub(crate) use self::extracted_artifacts::ExtractedArtifactDataHandle;
+use self::extracted_artifacts::ExtractedArtifacts;
```

Comment:
Sure, that does sound like a good practice. :+1:

Reply:
Since there are more changes to this file in the followup PRs, I'm going to tack this onto the end of #4042. I'll merge that (and #4041) into this PR prior to merging to `main`.